crawl-ref/source/monster-type.h

```diff
@@ -187,6 +187,13 @@ enum monster_type                      // env.mons[].type
     MONS_DIVINE_TOME,
     MONS_FROSTBOUND_TOME,
     MONS_STRANGE_MACHINE,
+    MONS_PHARAOH_ANT,
+    MONS_STEELBARB_WORM,
+    MONS_JOROGUMO,
+    MONS_BROODMOTHER,
+    MONS_CULICIVORA,
+    MONS_SUN_MOTH,
+    MONS_RADROACH,
```

Comment:
In principle, you should try to sort these into appropriate areas. (Jorogumo near harpy, etc.) It doesn't really matter (for several reasons), though.